spec/models/institution_spec.rb

```diff
@@ -185,4 +185,39 @@ def create_institution(institution, description)
       end
     end
   end
+
+  describe 'when exporting' do
+    let(:mapping) { described_class::CSV_CONVERTER_INFO }
+
+    # Pull the default CSV options to be used
+    default_options = Rails.application.config.csv_defaults[described_class.name] ||
+                      Rails.application.config.csv_defaults['generic']
+    load_options = default_options.each_with_object({}) { |(k, v), o| o[k.to_sym] = v; }
+
+    def check_attributes_from_records(rows, header_row)
+      described_class.find_each.with_index do |record, i|
+        attributes = {}
+
+        rows[i].each.with_index { |value, j| attributes[mapping[header_row[j]][:column]] = value }
+        csv_record = described_class.new(attributes)
+        csv_record.derive_dependent_columns if csv_record.respond_to?(:derive_dependent_columns)
+
+        csv_test_attributes = csv_record.attributes.except('id', 'version', 'created_at', 'updated_at', 'csv_row', 'version_id')
+        test_attributes = record.attributes.except('id', 'version', 'created_at', 'updated_at', 'csv_row', 'version_id')
+        test_attributes['ope'] = "\"#{test_attributes['ope']}\"" if test_attributes['ope']
+
+        expect(csv_test_attributes).to eq(test_attributes)
```

Comment:
I personally prefer to see the expectation declared directly within the spec example, but not that important.